projects/components/src/popover/popover-position-builder.ts

```diff
@@ -74,18 +78,25 @@ export class PopoverPositionBuilder {
 
       case PopoverRelativePositionLocation.BelowRightAligned:
         return new ConnectionPositionPair({ originX: 'end', originY: 'bottom' }, { overlayX: 'end', overlayY: 'top' });
+
       case PopoverRelativePositionLocation.LeftCentered:
         return new ConnectionPositionPair(
           { originX: 'start', originY: 'center' },
           { overlayX: 'end', overlayY: 'center' }
         );
+
+      case PopoverRelativePositionLocation.OverLeftAligned:
+        return new ConnectionPositionPair({ originX: 'start', originY: 'top' }, { overlayX: 'start', overlayY: 'top' });
```

Comment:
Is this different than aboveLeftAligned? The name seems to indicate the same thing to me.

Reply:
Yes. Above left aligned puts the bottom side of the popover container above the trigger element. So it makes the entire container above the trigger element